docs/faq.rst

```diff
@@ -376,8 +376,12 @@ which means you might need to update your existing components
 :ref:`signal handlers <topics-signals>` handlers or
 :ref:`spider middlewares <topics-spider-middleware>`) to make them work correctly.
 
-Alternatively, you can use the :function:scrapy.utils.decorators.subscriptable_dataclass``
-decorator, which adds the appropriate methods in order to make ``dataclass`` objects
+Alternatively, you can use the
+:func:`~scrapy.utils.decorators.subscriptable_dataclass` decorator:
+
+.. autofunction:: scrapy.utils.decorators.subscriptable_dataclass
+
+It adds the appropriate methods in order to make ``dataclass`` objects
```

Comment:
I changed `:function:` to `:func` and fixed the quoting.

The rest was purely :lipstick: aesthetic, so feel free to ignore: remove the import path from the reference display text, but display the API documentation right below.